app/assess/templates/macros/comments_summary.html

```diff
@@ -1,5 +1,5 @@
 {% macro comment_summary(comments, themes) %}
-<div class="govuk-!-margin-top-7" id="anchor">
+<div class="govuk-!-margin-top-7" id="comments">
```

Comment:
Should this be "comment" (see above)?

Reply:
I think this is correct, there are two anchors, `comment` for the "add comment" box form, and "comments" for the actual displaying of comments